src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1314,6 +1314,11 @@ def _match():
                 if m:
                     return m.group(1)
 
+            if self.daemon_type == 'rgw':
+                v = self.daemon_id.split('.')
+                if len(v) in [3, 4]:
+                    return '.'.join(v[0:2])
+
```

Comment:
:+1:   cherry-picked this into #35142, but can rebase the change out if you'd like to merge this first

Reply:
I think we can merge yours